python/lsst/dc2gen3/dc2gen3.py

```diff
@@ -20,6 +20,7 @@
 import lsst.log.utils
 from lsst.utils import doImport
 from lsst.obs.base.gen2to3 import ConvertRepoTask
+from lsst.obs.base.gen2to3 import CalibRepo
```

Comment:
There is no problem with importing multiple things from a single module in one line.

Reply:
Is combining them on the same line a preferred style?
Personally, I sometimes find it clearer to combine them, sometimes to separate them. In this case, I don't really have an opinion, although it does make diffs between versions very slightly easier to parse visually.

Reply:
```python
from lsst.obs.base.gen2to3 import (
    ConvertRepoTask,
    CalibRepo,
    SomethingElse,
)
```
also is clean with diffs and is the style used by black.

Reply:
That works for me.